.github/DISCUSSION_TEMPLATE/help.yml

```diff
@@ -1,25 +1,36 @@
----
 body:
-  - type: textarea
+  - type: markdown
+    attributes:
+      value: |
```

Comment:
This text uses three different words for the same thing: discussion, issue and thread.

Can we use one of them?

Reply:
thanks, fair point @JReinhold - resolved!